### ISSUE TITLE
Disabling className removes default class form-control

### ISSUE THREAD

**Issue by KuenzelIT** [kevinchappell/formBuilder#544]
Hey,

is it possible to disable (or rather hide) the `className` attribute but keep the default class `form-control` for every field?

Thanks!

**Comment by PhuAudi** [comment]
Did you solve the problem?
I also have problems like you :(

**Comment by KuenzelIT** [comment]
I ignored this for now, but if I had to solve this right now, I would disable the className, catch the `fieldAdded` event and add the class `form-control` to every input which doesn't already have it.

**Comment by PhuAudi** [comment]
I will try to make your way.
Thanks you 👍

**Comment by PhuAudi** [comment]
I solved this problem as follows:

<style>
    .form-group.className-wrap, .form-group.name-wrap, .form-group.access-wrap, .form-group.value-wrap {
        display: none !important;
    }
</style>

**Comment by KuenzelIT** [comment]
Nice solution!

**Comment by rssluca** [comment]
Hi, I am looking for a cleaner solution as well. Is there a way to set the default class for fields?

**Comment by rssluca** [comment]
I resolved adding the attribute in the control file build call.

in controls/text.js
```
  build() {
    this.config['className'] ='form-control'; // Add this line
    return this.markup('input', null, this.config);
  }
```
in textarea.js

```
  build() {
    this.config['className'] ='form-control'; // Add this line
    let {value = '', ...attrs} = this.config;
    this.field = this.markup('textarea', this.parsedHtml(value), attrs);
    return this.field;
  }
```

**Comment by kevinchappell** [comment]
fixed in 2.10.0
